numpy/lib/_format_impl.py

```diff
@@ -700,6 +708,7 @@ def _read_array_header(fp, version, max_header_size=_MAX_HEADER_SIZE):
     return d['shape'], d['fortran_order'], dtype
 
 
+
```

Comment:
Please revert this, the linter should actually prefer 2 blank lines.  (The lint CI job is failing, although there are some other failures that may be spurious.)